### PR TITLE
Rsquared fix plus more docs

### PR DESCRIPTION
**Pull request by ablaom**
Continuation of #683

### CODE REVIEW

**Comment by ablaom** [comment]
@rikhuijzer Be great if you can look over this. Thanks for getting me started.

File: src/measures/meta_utilities.jl [hunk 1]
```diff
@@ -90,7 +90,73 @@ end
...
+  has the `instances = ["macro_f1score", "micro_f1score",
+  "multiclass_f1score"]`, where `micro_f1score =
+  MulticlassFScore(average=micro_avg)`, etc.  Default is `String[]`.
+
+- `aggregation`: Aggregation method for measurements, typically
+        `Mean()` (for, e.g., mean absolute error) or `Sum()` (for number
+    of true positives). Default is `Mean()`. Must subtype
+    `StatisticalTraits.AggregationMode`. It is used to:
+
+   - aggregrate measurements in resampling (eg, cross-validiation)
```
Comment (rikhuijzer) [inline src/measures/meta_utilities.jl:136]:
```suggestion
   - aggregrate measurements in resampling (e.g., cross-validation)
```

File: src/measures/meta_utilities.jl [hunk 2]
```diff
@@ -90,7 +90,73 @@ end
...
+- `supports_weights`: Whether the measure can be called with
+  per-observation weights `w`, as in `l2(ŷ, y, w)`. Default is `true`.
+
+- `supports_class_weights`: Whether the measure can be called with a
+  class weight dictionary `w`, as in `micro_f1score(ŷ, y, w)`. Default
+  is `true`. Default is `false`.
+
+- `human_name`: Ordinary name of measure. Used in the full
+  auto-generated docstring, which begins "A measure type for
+  $human_name ...". Eg, the `human_name` for `TruePositive` is `number
```
Comment (rikhuijzer) [inline src/measures/meta_utilities.jl:151]:
```suggestion
  \$human_name ...". Eg, the `human_name` for `TruePositive` is `number
```

Without this, the output is messed up:

image

File: src/measures/meta_utilities.jl [hunk 3]
```diff
@@ -90,7 +90,73 @@ end
...
+
+- `aggregation`: Aggregation method for measurements, typically
+        `Mean()` (for, e.g., mean absolute error) or `Sum()` (for number
+    of true positives). Default is `Mean()`. Must subtype
+    `StatisticalTraits.AggregationMode`. It is used to:
+
+   - aggregrate measurements in resampling (eg, cross-validiation)
+
+   - aggregating per-observation measurements returned by `single` in
+    the fallback definition of `call` for `Unaggregated` measures
```
Comment (rikhuijzer) [inline src/measures/meta_utilities.jl:139]:
```suggestion
     the fallback definition of `call` for `Unaggregated` measures
```
Adding one space here will fix indentation problems in the REPL output. Currently, "the fallback" is incorrectly put on a new line. This is a very minute detail. Probably best ignored.

**Review by rikhuijzer — COMMENTED** [review]
Thanks for improving the docs! Much clearer now

File: src/measures/meta_utilities.jl (hunk 2 above)
Comment (ablaom) [reply src/measures/meta_utilities.jl:151]: Great catch! I forgot to check this 🙄

**Comment by ablaom** [comment]
@rikhuijzer Thanks very much!

**Pull request merged**